lib/routes/home/widgets/payments_list/dialog/closed_channel_payment_details.dart

```diff
@@ -16,62 +18,63 @@ class ClosedChannelPaymentDetailsWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final themeData = Theme.of(context);
     final texts = context.texts();
+    return FutureBuilder<Config>(
+      future: _fetchConfig(),
+      builder: (BuildContext context, AsyncSnapshot<Config> snapshot) {
+        if (snapshot.hasData) {
+          final blockExplorer = snapshot.data!.defaultMempoolUrl;
+          if (paymentMinutiae.status == sdk.PaymentStatus.Complete) {
+            return Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                RichText(
+                  text: TextSpan(
+                    style: themeData.dialogTheme.contentTextStyle,
+                    text: texts.payment_details_dialog_closed_channel_local_wallet,
+                  ),
+                ),
+                if (paymentMinutiae.paymentType == sdk.PaymentType.ClosedChannel &&
+                    paymentMinutiae.fundingTxid != null) ...[
+                  TxWidget(
+                    txURL: "$blockExplorer/tx/${paymentMinutiae.closingTxid!}",
+                    txID: paymentMinutiae.closingTxid!,
+                  ),
+                ],
+              ],
+            );
+          }
 
-    if (paymentMinutiae.status == PaymentStatus.Complete) {
-      return Column(
-        mainAxisSize: MainAxisSize.min,
-        children: [
-          RichText(
-            text: TextSpan(
-              style: themeData.dialogTheme.contentTextStyle,
-              text: texts.payment_details_dialog_closed_channel_local_wallet,
-            ),
-          ),
-          // TODO waiting for closeChannelTxUrl and closeChannelTx
-          const TxWidget(
-            txURL: "",
-            txID: "",
-          ),
-        ],
-      );
-    }
+          String estimation = texts.payment_details_dialog_closed_channel_transfer_no_estimation;
 
-    int lockHeight = 0; // TODO pendingExpirationHeight
-    double hoursToUnlock = 0.0; // TODO hoursToExpire
-
-    int roundedHoursToUnlock = hoursToUnlock.round();
-    String hoursToUnlockStr = roundedHoursToUnlock > 1
-        ? texts.payment_details_dialog_closed_channel_about_hours(
-            roundedHoursToUnlock.toString(),
-          )
-        : texts.payment_details_dialog_closed_channel_about_hour;
-    String estimation = lockHeight > 0 && hoursToUnlock > 0
-        ? texts.payment_details_dialog_closed_channel_transfer_estimation(
-            lockHeight,
-            hoursToUnlockStr,
-          )
-        : texts.payment_details_dialog_closed_channel_transfer_no_estimation;
```

Comment:
Can we add // TODO comments here so we don't forget to integrate transfer estimation information?

@roeierez @ok300 Do we have any plans to add these variables to ClosedChannelPaymentDetails or introduce a transfer estimation function on the SDK?